_blog.yml

```
@@ -6575,4 +6575,16 @@
     - llm
     - qwen
     - flux
-    - text-to-image
\ No newline at end of file
+    - text-to-image
```

```suggestion
    - text-to-image
    - image-to-image
    - text-to-video
    - image-to-video
```
how about we addd these too? i2i for kontext & qwen edit, t2v & i2v for wan/ltx (though only discussed briefly in the blog, but i2v is a very popular task)